scripts/build.js

```diff
@@ -84,6 +90,19 @@ function build(options) {
 }
 
 /**
+ * Resolve node-gyp command to invoke
+ *
+ * @api private
+ */
+function resolveNodeGyp() {
```

Comment:
No and that's why i removed my changes in https://github.com/sass/node-sass/pull/1777/commits/5021dff1df351c5a3c4f3c81cbfb65c09ec23aa2 . I just kept the resolution of `node-gyp` in separate function for easy readability.